openfisca_core/periods.py

```diff
@@ -673,6 +673,29 @@ def unit(self):
         return self[0]
 
 
+    # Reference periods
+
+    @property
+    def last_3_months(self):
+        return self.this_month.start.period('month', 3).offset(-3)
+
+    @property
+    def last_month(self):
+        return self.this_month.offset(-1)
+
+    @property
+    def n_2(self):
```

Comment:
:+1: Matti, the idea is really to give shortcuts for periods used in the vast majority of formulas, so I don't think putting flexibility here would be relevant. 

About `n_2`, let's think about a more transparent term.  I thought about `2_years_ago`, but it's not really what we mean. If we are in April 2015, two years ago is April 2013, not 2013. Would  `n_2_year` be more explicit ? 

I also wondered if it was a good idea to put it in core, but the `period.last_year` notation seemed pretty intuitive to me, and I didn't have a better idea.